gen/top_gen.ml

```diff
@@ -168,11 +167,11 @@ module U = TopUtils.Make(O)(Comp)
           let r,init,i,st =
             Comp.emit_load_one st p init loc in
           Some r,init,i,st
-        else
+        else begin
```

Comment:
is it necessary to add this begin ... end catch?

Reply:
Removed.